docs/technical/the-great-stores-migration.md

```diff
@@ -0,0 +1,160 @@
+# Organizing Desktop's Store Interactions For Great Good
+
+I've been pondering on what to do with our `AppStore` and other stores given
```

Comment:
are these concerns and examples meant to be generalized to stores like `git-store`, `issue-store`, and on? there are a lot of `*-store.ts` files, so i want to make sure i understand the scope of this document as it currently stands.

Reply:
👍 I've mostly been focused on `AppStore` initially but I can expand this to some gut feels about those and how to organize them.